wordpress-mobile/WordPress-Android
# Move insights up and down in the list

Fixes #8953 
This is related to the FluxC PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1088

This PR adds an overflow menu to each Title on the Insights tab. This menu contains button to move the item up, down or remove it from the list. 

To test:
* Open Stats
* You see only Latest post summary, Today stats and Most popular block in Stats
* You see an button in each title
* Clicking the button opens a Popup menu
* The popup menu contains 3 items - `Move up`, `Move down` and `Remove`
  * The first item doesn't contain the `Move up` action
  * The last item doesn't contain the `Move down` action
* Clicking the actions moves the block up/down/removes it from the list
* Adding blocks is not yet implemented so be careful with testing it
